xml/planning-planning-objectstorage-creating_pac_cluster.xml

```diff
@@ -58,9 +58,9 @@
          - swift-client</screen>
     <important>
      <para>
-      Do not change the name of the cluster <literal>swpac</literal> as it may
-      conflict with an existing cluster. Use a name such as
-      <literal>swpac1</literal>, <literal>swpac2</literal>, or
+      Do not change the name of the cluster <literal>swpac</literal> to ensure
+      that it remains unique among clusters. Use names for its servers such as
```

Comment:
"Use a name for servers such as ..."

Reply:
Definitely needs to stay as-is for accuracy. The admin will be defining multiple names for its servers and they should be addressed by their plural.